packages/rum-core/src/bootstrap.js

```diff
@@ -43,3 +43,23 @@ export function bootstrap() {
 
   return enabled
 }
+
+export function bootstrapPerf() {
+  if (document.visibilityState === 'hidden') {
+    state.lastHidden = {
```

Comment:
can you change it to `state.lastHiddenTime` ? Would be easier to understand and we don't end up with another depth.